pkg/controller/snapshot_controller.go

```diff
@@ -735,22 +731,12 @@ func (ctrl *csiSnapshotController) GetClassFromVolumeSnapshot(snapshot *crdv1.Vo
 	glog.V(5).Infof("getClassFromVolumeSnapshot [%s]: VolumeSnapshotClassName [%s]", snapshot.Name, className)
 
 	if len(className) > 0 {
-		obj, found, err := ctrl.classStore.GetByKey(className)
-		if found {
-			class, ok := obj.(*crdv1.VolumeSnapshotClass)
-			if ok {
-				return class, nil
-			}
-		}
-		class, err := ctrl.clientset.VolumesnapshotV1alpha1().VolumeSnapshotClasses().Get(className, metav1.GetOptions{})
+		class, err := ctrl.classLister.Get(className)
 		if err != nil {
-			glog.Errorf("failed to retrieve snapshot class %s from the API server: %q", className, err)
-			return nil, fmt.Errorf("failed to retrieve snapshot class %s from the API server: %q", className, err)
-		}
-		_, updateErr := ctrl.storeClassUpdate(class)
```

Comment:
Why removing this?

Reply:
ok, I agree to remove this. but maybe in a separate PR?